src/components/actions/runActionRoute.ts

```diff
@@ -15,6 +15,10 @@ export const routeRunAction = async (request: any, reply: any) => {
 }
 
 export const routeGetApplicationData = async (request: any, reply: any) => {
-  const { applicationId } = combineRequestParams(request, 'camel')
-  reply.send(await getApplicationData({ applicationId: Number(applicationId) }))
+  const { applicationId, reviewId } = combineRequestParams(request, 'camel')
+  const appDataParams: { applicationId: number; reviewId?: number } = {
+    applicationId: Number(applicationId),
+  }
+  if (reviewId) appDataParams.reviewId = Number(reviewId)
+  reply.send(await getApplicationData(appDataParams))
```

Comment:
Is this only used by template builder ? (I would imagine this was added to see full application data for review via postman ?)

Reply:
> Is this only used by template builder?

It's not even used by the template builder, it's purely a dev tool (and I think it should stay that way). We can make more specific endpoints (such as "Preview" and "Expire") for actual tasks we need the app to do. The "/run-action" endpoint is a very blunt instrument and shouldn't be widely available.

> (I would imagine this was added to see full application data for review via postman ?)

Yes, exactly.

Reply:
> It's not even used by the template builder

There was a link to it from application builder, i think it's used for application data in actions